src/data_pipeline.py

```diff
@@ -129,7 +129,7 @@ def main():
     ####################################################################################
     print("2. Identifying subplant IDs")
     # GTN ratios are saved for reloading, as this is computationally intensive
-    if not os.path.exists(outputs_folder(f"{year}/subplant_crosswalk.csv")):
+    if not os.path.exists(outputs_folder(f"{year}/subplant_crosswalk_{year}.csv")):
```

Comment:
Since this is faster now, should we re-run with every pipeline run?

Reply:
Good point, probably would be good practice.